tests/e2e/install/cockroachdb_helm_e2e_test.go

```diff
@@ -639,3 +600,38 @@ func testWALFailoverExistingCluster(t *testing.T, additionalValues map[string]st
 		testutil.RequireClusterToBeReadyEventuallyTimeout(t, crdbCluster, 600*time.Second)
 	}
 }
+
+func getHelmValues(inputValues map[string]string) map[string]string {
+	defaultValues := map[string]string{
+		// Override the persistent storage size to 1Gi so that we do not run out of space.
+		"storage.persistentVolume.size": "1Gi",
+	}
+
+	for k, v := range defaultValues {
+		inputValues[k] = v
```

Comment:
I would expect this function to merge the default values with the passed values, using the passed values as overrides. This way you would be able to set the storage size to something other than 1Gi.
maybe change the function to something like:
```go
for k, v:= range inputValues {
	defaultValues[k] = v
}
return defaultValues
```

Reply:
I had actually intended to override the input values. I could maybe update the variable/function names to make this clear.
I don't see a reason why a test would need >1Gi persistent storage. If it comes to that, will change it later.